### DISCUSSION TITLE
owo, oworun

### DISCUSSION THREAD

**Discussion by OneVi** [ahihiyou20/discord-selfbot-owo-bot#5] [Ideas]
can you add function with sending owo and oworun owopup owopiku?

**Comment by ahihiyou20** [comment]
Eh i don't think it's related to the Selfbot

**Reply by OneVi** [reply]
okay, i have another question what mean wbm in your code?

**Reply by ahihiyou20** [reply]
> okay, i have another question what mean wbm in your code?

The cooldown beetween each hunt, battle command
